ntbs-service/Models/RiskFactorDrugs.cs

```diff
@@ -1,8 +0,0 @@
-using Microsoft.EntityFrameworkCore;

```

Comment:
Don't need separate models for these, we use type to make clear which is which. Also, they are currently mapped to different tables anyway.

Reply:
Yes, they all look something like
```x.OwnsOne(c => c.RiskFactorDrugs , rf => {
                        rf.Property(e => e.Status).HasConversion(statusEnumConverter);
                        rf.Property(e => e.Type).HasConversion(riskFactorEnumConverter).HasDefaultValue(RiskFactorType.Drugs);
                        rf.ToTable("RiskFactorDrugs");
                    });```